### PR TITLE
Make LEAP dask workers be not pre-emptible

### PR DESCRIPTION
**Pull request by yuvipanda**
Possible way to reduce confounding factors for
https://github.com/2i2c-org/infrastructure/issues/2396

### CODE REVIEW

**Review by consideRatio — APPROVED** [review]
LGTM!

Can you add a comment to the leap.tfvars about deviating from the default of `preemptible: true` to `false` for leap? I think its good if this intentional workaround stands out

File: terraform/gcp/projects/leap.tfvars [hunk 1]
```diff
@@ -84,6 +84,7 @@ dask_nodes = {
   "medium" : {
     min : 0,
     max : 200,
+    preemptible: false,
```
Comment (pnasrat) [inline terraform/gcp/projects/leap.tfvars:87]: Can you add a comment linking back to the logic for this change

**Review by pnasrat — CHANGES_REQUESTED** [review]
minor documentation nit will approve once fixed

File: terraform/gcp/projects/leap.tfvars (hunk 1 above)
Comment (pnasrat) [reply terraform/gcp/projects/leap.tfvars:87]: oh I see @consideRatio and I collided on this review with same feedback!

**Comment by consideRatio** [comment]
Btw they may very well need a cloud quota increase for on-demand CPU now. pre-emptible/spot CPU quotas are separate from on-demand I believe, so they may run out on on-demand now that its the only kind of CPU used.

**Comment by yuvipanda** [comment]
Am also going to wait for the LEAP credits issue to be resolved before deploying this.

**Comment by yuvipanda** [comment]
Leap credits are done now

**Review by pnasrat — APPROVED** [review]
lgtm

**Pull request merged**